src/video.js

```diff
@@ -11,6 +11,16 @@ const handleStop = (event) => {
 };
 
 const videoOptions = () => {
+  // TODO, These values are the same the MediaRecorder should have by default.
+  // We identified that in recent Safari versions the defaults were not being respected
+  // thus we have to add them here to ensure we controll the bitrate and keep file sizes resonable.
```

Comment:
Some minor typos here.
```suggestion
  // thus we have to add them here to ensure we control the bitrate and keep file sizes reasonable.
```